src/postgres/include/c.h

```diff
@@ -289,17 +289,6 @@ typedef unsigned long long int uint64;
 #define UINT64_FORMAT "%" INT64_MODIFIER "u"
 
 /*
- * 128-bit signed and unsigned integers
- *		There currently is only a limited support for the type. E.g. 128bit
- *		literals and snprintf are not supported; but math is.
- */
-#if defined(PG_INT128_TYPE)
-#define HAVE_INT128
-typedef PG_INT128_TYPE int128;
-typedef unsigned PG_INT128_TYPE uint128;
-#endif
-
-/*
```

Comment:
Heh, the only reason its useful is because we can then just teach the automatic code generation about this :)

Specifically, we should be able to take care of the whole deal by adding:

```echo "#undef PG_INT128_TYPE" >> ./src/postgres/include/pg_config.h```

after this line: https://github.com/lfittl/libpg_query/blob/47e6956c0d1182fcce4a23fd04ee3626887c9f10/Makefile#L70

Happy to take care of adding it there post-merge though, since you need a few more tools to run "extract_source" :)